test/test12.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=build/
+if build/marc --debug output=tmp.bin input/test2.opl; then
```

Comment:
This will fail .. did you mean --output?

Reply:
I did mean --output. Thanks for catching that!